### PR TITLE
[logging] log when in a test after 60 seconds and once every 5 after that.

### PR DESCRIPTION
**Pull request by rexhoffman**
log when in a test after 60 seconds and once every 5 after that.

Log lines will look like:

```
        PASS [  10.027s]                                   language-e2e-testsuite  tests::peer_to_peer::many_to_one_peer_to_peer
/opt/git/diem/target/debug/deps/generated_files-9d775a91e3f25118::test_that_generated_file_are_up_to_date_in_git elapsed time is 90
```

https://github.com/diem/diem/runs/3256231330#step:6:2195

### CODE REVIEW

File: nextest/runner/src/runner.rs [hunk 1]
```diff
@@ -314,9 +315,20 @@ impl<'list> TestRunner<'list> {
...
 
-        // TODO: timeout/kill logic
+        let now = Instant::now();
 
-        let output = handle.into_output()?;
+        let output = loop {
+            let result = handle.try_wait();
+            println!("{:?} elapsed time is {}", &test, now.elapsed().as_secs());
+            if let Ok(None) = result {
+                sleep(Duration::new(2, 0));
```
Comment (sausagee) [inline nextest/runner/src/runner.rs:324]: Do we want to up this limit?  For reference, there are ~100 tests take over 20sec to complete (see histogram https://ci-artifacts.diem.com/testhistory/diem/diem/auto/ci-test.yml/1091973229/report/index.html#graph).

**Review by sausagee — APPROVED** [review]
Just want to point out the design consideration in the sleep time.  Otherwise, the change looks good.

**Comment by sausagee** [comment]
PR summary doesn't seem to match the code change.  Maybe typo?

**Comment by sausagee** [comment]
This looks like a pick from #97.  Maybe just cherry-pick it?

File: nextest/runner/src/runner.rs (hunk 1 above)
Comment (rexhoffman) [reply nextest/runner/src/runner.rs:324]: huh somehow got an older revision on this.  will update.

Comment (rexhoffman) [reply nextest/runner/src/runner.rs]: resolved

**Pull request merged**